pkg/resource/deploy/deploytest/pluginhost.go

```diff
@@ -183,6 +183,8 @@ func wrapProviderWithGrpc(provider plugin.Provider) (plugin.Provider, io.Closer,
 }
 
 type hostEngine struct {
+	pulumirpc.UnsafeEngineServer // opt out of forward compat
```

Comment:
My thought was that if we're adding a new RPC to the engine, we should implement it at the same time here.

Same with the Go language server, which seems like it would likely be the last to move out of `pulumi/pulumi`.

Though if we want consistency, we could change this to `UnimplementedEngineServer` and likewise for the language server.

Reply:
Yup that's reasonable. I was going to say I'd like to keep compile errors in pu/pu for when adding new methods, but mostly around "engine" stuff rather than e.g. the language servers.